seeders/20220726113045-users-seed-file.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
         email: `user${i + 2}@example.com`,
         password: bcrypt.hashSync('12345678', 10),
         name: faker.lorem.words(2),
-        avatar: 'https://joeschmoe.io/api/v1/random',
+        avatar: `https://i.pravatar.cc/300?img=${i + 46}`,
```

Comment:
Please consider to add a comment to note that the avatar api only valid while img= 1 to 70.

Reply:
Comment of avatar API added.